identity/app/services/ProfileRedirectService.scala

```diff
@@ -22,8 +22,8 @@ sealed abstract class ProfileRedirect(val url: String) {
   def isAllowedFrom(url: String): Boolean
 }
 
-case object RedirectToEmailValidationFromEmailPrefs extends ProfileRedirect("/verify-email") {
-  override def isAllowedFrom(url: String): Boolean = url contains "email-prefs"
+case object RedirectToEmailValidationFromEmailPrefsOrConsentJourney extends ProfileRedirect("/verify-email") {
+  override def isAllowedFrom(url: String): Boolean = (url contains "email-prefs") || (url contains "consents")
```

Comment:
can this be a bit more specific - at least use `startsWith` instead of contains?

Reply:
for both email prefs and consents